connect/zypper.go

```diff
@@ -120,12 +142,48 @@ func removeService(serviceName string) error {
 	return removeServiceCredentials(serviceName)
 }
 
+func enableServiceAutorefresh(serviceName string) error {
+	args := []string{"--non-interactive", "modifyservice", "-r", serviceName}
+	_, err := zypperRun(args, true, []int{zypperOK})
+	return err
+}
+
+func refreshService(serviceName string) error {
+	args := []string{"--non-interactive", "refs", serviceName}
+	_, err := zypperRun(args, true, []int{zypperOK})
+	return err
+}
+
 func refreshAllServices() error {
 	args := []string{"--non-interactive", "refs"}
 	_, err := zypperRun(args, true, []int{zypperOK})
 	return err
 }
 
+func installReleasePackage(identifier string) error {
+	if identifier == "" {
+		return nil
+	}
+	args := []string{"rpm", "-q", identifier + "-release"}
+	if _, err := execute(args, false, nil); err != nil {
```

Comment:
this is wrong - it should return if there is no error, ie the rpm is already installed

Reply:
done

Reply:
ah, right, the go-style error handling is very error prone wrt negative tests. maybe rename this to `ok` to make it clear what we're up to?

Reply:
no it was my mistake reading the ruby "unless" statement. I added a comment with the latest push

Reply:
`unless` is very error prone as well :o)
people are not good at inverse logic.